src/views/statutory/EditPositionModal.vue

```diff
@@ -128,6 +140,17 @@ export default {
     this.dates.starts = this.position.starts = new Date(this.position.starts)
     this.dates.ends = this.position.ends = new Date(this.position.ends)
     this.dates.ends_force = this.position.ends_force = new Date(this.position.ends_force)
+
+    this.axios.get(this.services['oms-core-elixir'] + '/bodies').then((response) => {
```

Comment:
and the fetching is not required here in modal if we can fetch all of them in PositionsList